README.rst

```diff
@@ -3,9 +3,13 @@ COUNTER Code of Practice Release 5
 
 This repository contains the source for the `COUNTER <https://www.projectcounter.org/>`_ Code of Practice Release 5.
 
-There are three branches:
+There are multiple branches:
 
-* `5.0.2 <https://github.com/Project-Counter/cop5/tree/5.0.2>`_: Release 5.0.2 (published 28 September 2021) will become the current Code of Practice and the requirement for COUNTER compliance effective from January 2022.
+* `5.1 <https://github.com/Project-Counter/cop5/tree/5.0.3>`_: Release 5.1 (published April 2023) will become the current Code of Practice and the requirement for COUNTER compliance effective from January 2025.
+
+* `5.0.3 <https://github.com/Project-Counter/cop5/tree/5.0.3>`_: Release 5.0.3 (published April 2023) will become the current Code of Practice and the requirement for COUNTER compliance effective from January 2024.
```

Comment:
5.0.3 is a bugfix only release, therefore it should become effective when it is published.